problems/removeEvenStrings.js

```diff
@@ -5,9 +5,9 @@
 * @param {string[]} arr - The input array of strings
 * @returns {string[]} - Returns the strings in arr that have an odd number of characters
 */
-
+//This one was tough
 function removeEvenStrings() {
-
-}
-
+    let arr = [];
```

Comment:
The arr needs to come in as a parameter. It's passed as an argument. 
You'll also need to use filter to get this working